### PR TITLE
Adjust auth proxy SAR to be more specific

### PR DESCRIPTION
**Pull request by csibbitt**
Actually-Dependz-On https://github.com/infrawatch/service-telemetry-operator/pull/553/ but they are cross-dependent

### CODE REVIEW

**Comment by csibbitt** [comment]
recheck

File: deploy/olm-catalog/smart-gateway-operator/manifests/smart-gateway-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -185,14 +185,6 @@ spec:
           - update
           - patch
           - watch
-        - apiGroups:
-          - ""
-          resources:
-          - namespaces
```
Comment (leifmadsen) [inline deploy/olm-catalog/smart-gateway-operator/manifests/smart-gateway-operator.clusterserviceversion.yaml:191]: This was a result of our previous attempt at alignment with openshift-monitoring?

**Review by leifmadsen — APPROVED** [review]

File: deploy/olm-catalog/smart-gateway-operator/manifests/smart-gateway-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (csibbitt) [reply deploy/olm-catalog/smart-gateway-operator/manifests/smart-gateway-operator.clusterserviceversion.yaml:191]: Yes.

**Comment by csibbitt** [comment]
recheck

**Pull request merged**